CharacterCreation/01_Races.txt

```diff
@@ -1,49 +1,62 @@
-//============================================================
+==============================
 Human
-Society:
-Humans are the most common sentient species in ECSC. While most belong to the 
+==============================
+
+===== Society =====
+
+    Humans are the most common sentient species in ECSC. While most belong to the 
 empire that is centered near their home planet, many other cultures and sub 
 cultures dot the surrounding systems ranging from corporations to patriarchies 
 to pirate anarchies. 
+
 Type:		Organic
 Size:		Medium
 Speed:		Normal
 Starting Stat Points: 42
 Stat Modifiers:	
 +2 stat points to spend at level 1
-Language: 	Emperial Common, +1 dialect
-Racial Traits:	
+Language: 	Imperial Common, +1 dialect
+
+===== Racial Traits =====	
+
 None
 
-//============================================================
+==============================
 Tundari
+==============================
+
 Society: Tribal system, lead by a council of elders.

```

Comment:
Blocker - Trash. Line 28 is redundant with line 48-50 and non-compliant with style.

Reply:
Took out the trash.

Reply:
I still see it in the Files Changed view.